CMakeLists.txt

```diff
@@ -237,7 +247,7 @@ include(CPack)
 message(
     "\n====================|  Spicy Build Summary  |===================="
     "\n"
-    "\nVersion:               ${SPICY_VERSION_LONG}"
+    "\nVersion:               ${SPICY_VERSION} ${SPICY_COMMIT}"
```

Comment:
Do we need this change? `SPICY_VERSION_LONG` seems to be what we want?

Reply:
Good catch, this was a leftover from massaging the existing code to the form of this patch.